Source/WebCore/dom/InternalObserver.cpp

```diff
@@ -28,8 +28,9 @@
 
 #include "ContextDestructionObserverInlines.h"
 #include "JSDOMExceptionHandling.h"
+#include <JavaScriptCore/Exception.h>
 #include <JavaScriptCore/JSCJSValue.h>
-#include <JavaScriptCore/JSGlobalObjectInlines.h>
```

Comment:
Why isn’t this needed any more? Seems really surprising this file doesn’t call any functions that have inline bodies in that header.

Reply:
It looks like we can just include JSGlobalObject.h instead of JSGlobalObjectInlines.h.